src/main/java/com/gmail/val59000mc/scenarios/scenariolisteners/RandomItemsListener.java

```diff
@@ -0,0 +1,64 @@
+package com.gmail.val59000mc.scenarios.scenariolisteners;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.inventory.ItemStack;
+
+import com.gmail.val59000mc.scenarios.ScenarioListener;
+import com.gmail.val59000mc.utils.RandomUtils;
+
+public class RandomItemsListener extends ScenarioListener{
+	
+	private List<Material> materialitems;
```

Comment:
Rename to "materialItems" to follow java coding conventions.
Or maybe just "items".